pinniped-components/tanzu-auth-controller-manager/controllers/v3_cascade_controller_test.go

```diff
@@ -51,9 +62,24 @@ var _ = Describe("Controller", func() {
 			},
 			Type: clusterBootstrapManagedSecret,
 		}
+
+		// This Secret is to configure Pinniped on the management cluster, so this controller should never
+		// edit this Secret, since it is not responsible for configuring Pinniped on management clusters.
+		managementClusterPinnipedCBSecret = &corev1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: pinnipedNamespace,
+				Name:      fmt.Sprintf("%s-pinniped.tanzu.vmware.com-package", managementCluster.Name),
+				Labels: map[string]string{
+					packageNameLabel:    testPinnipedLabel,
+					tkgClusterNameLabel: managementCluster.Name,
+				},
+			},
+			Data: map[string][]byte{tkgDataValueFieldName: []byte("should-not-be-edited")},
+			Type: clusterBootstrapManagedSecret,
+		}
 	})
 
-	Context("ClusterBootstrap Secret", func() {
+	Context("pinniped-info configmap does not exist", func() {
```

Comment:
The `pinniped-info` configmap  is a contract for several use cases:
- the WCP (TKGs) Supervisor cluster provides it, to provide consistency with our usual Management (TKGm) clusters. 
- the tanzu cli knows to look for this configmap

Reply:
👍